src/restart.F

```diff
@@ -4591,6 +4591,7 @@ subroutine writer(numi,numj,numk1,numk2,nxr,nyr,var,aname,           &
 #ifdef MPI
     rf2:  IF( restart_filetype.eq.3 )THEN
 
+      print *,'writer: before writer2'
```

Comment:
@johnmauff Is this line still needed?

Reply:
Thanks for that catch.  I just deleted it.

Reply:
Thanks @johnmauff